src/views/Quiz/js/quiz-questions.js

```diff
@@ -70,6 +74,35 @@ export default {
           return 'D'
       }
     },
+    getTimeLimit () {
+      this.fetchStudentQuizTimeLimit({
+        data: {
+          batchCode: this.currentUser.batchCode,
+          quizId: this.$route.params.quizId
+        },
+        callback: this.successFetchingTimeLimit,
+        fail: this.failedFetchingTimeLimit
+      })
+    },
+    successFetchingTimeLimit (response) {
+      this.quizDetail.timeLimit = response
+      this.isLoading = false
+    },
+    failedFetchingTimeLimit () {
+      this.toast({
+        data: {
+          message: 'Something went wrong',
```

Comment:
better to have explicit message like `failed to start quiz`, and redirect to quiz detail page.
it's better to create an issue to change this though